lib/reek/examiner.rb

```diff
@@ -113,14 +116,10 @@ def syntax_tree
     end
 
     def examine_tree
-      return unless syntax_tree
+      return [] unless syntax_tree
```

Comment:
Can we add a comnent here that explains that this happens on a source file with no code? I do not consider this obvious.

Reply:
Agreed.

Reply:
I refactored the code so this whole line is gone :tada:.